### DISCUSSION TITLE
PCL数据存储到本体目录

### DISCUSSION THREAD

**Discussion by 1602279568** [Meloong-Git/PCL#2047] [提问] [answered]
PCL的数据能不能存储到本体目录啊，学校的冰点还原每次重启都会把数据给擦除掉...

**Comment by Chiloven945** [comment]
PCL的部分数据储存在注册表，而且请不要乱交Categories

**Comment by WForst-Breeze** [comment, marked as answer]
PCL2 会在注册表中储存部分隐私设置，包括隐藏主题、账户信息、Java 路径、游戏文件夹路径等。这是为了数据安全而做的刻意的设计，您可以通过导出这些数据来手动恢复除隐藏主题以外的所有数据。

**Reply by iPodtouch6557** [reply]
以外，不是意外

**Reply by E0x72-21** [reply]
如何手动导出备份以及迁移至其他电脑呢？（比如我有时候要换机）

**Reply by allMagicNB** [reply]
> 如何手动导出备份以及迁移至其他电脑呢？（比如我有时候要换机）

去帮助库里边搜备份

**Reply by witherINminecraft** [reply]
> > 如何手动导出备份以及迁移至其他电脑呢？（比如我有时候要换机）
> 
> 去帮助库里边搜备份

您好，能说一下具体步骤吗？(这很重要

**Reply by Joker2184** [reply]
> 如何手动导出备份以及迁移至其他电脑呢？（比如我有时候要换机）

注：导出备份 隐藏主题相关的 无法迁移 
只能在本机使用

**Reply by WForst-Breeze** [reply]
> 您好，能说一下具体步骤吗？(这很重要

在 更多-帮助 页面的搜索框搜索 “备份” 即可找到备份启动器设置的条目，依照该页面中的描述备份即可

**Comment by E0x72-21** [comment]
> > 如何手动导出备份以及迁移至其他电脑呢？（比如我有时候要换机）
> 
> 去帮助库里边搜备份

感谢

**Comment by Joker2184** [comment]
[刻意的程序设计]

**Comment by wuliaodexiaoluo** [comment]
>如何手动导出备份以及迁移至其他电脑呢？（比如我有时候要换机）

PCL 的备份只能用于在没有更换关键硬件的设备上，且不能切换到新设备，因为这样会导致产生的密钥发生变化，进而导致无法解密数据
